.gitignore

```diff
@@ -70,3 +70,7 @@ storybook-static/
 test-results.json
 
 .idea/
+
+# OS-X
+.DS_Store
+src/.DS_Store
```

Comment:
You can just add this to your global git ignore https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Then just add `.DS_Store` and it should ignore it in every repo everywhere

Reply:
remove this from the repo .gitignore

Reply:
Thanks for that. I didn't know about it.